src/shady-css/ast-iterator.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt The complete set of authors may be found
+ * at http://polymer.github.io/AUTHORS.txt The complete set of contributors may
+ * be found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by
+ * Google as part of the polymer project is also subject to an additional IP
+ * rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+import * as util from 'util';
+
+import {Node, nodeType} from './common';
+
+export function* iterateOverAst(node: Node): Iterable<Node> {
```

Comment:
It's exported from the project as this name, and there might be other things we'd want to iterate over...